### PR TITLE
Update: Ultra Intro change

### PR DESCRIPTION
**Pull request by OjesviPandita**
Changing introduction to more generalised format

### CODE REVIEW

**Comment by thekirankumar** [comment]
@gauthamsrinivas7 is reviewing this

**Comment by thekirankumar** [comment]
Whats the difference between this PR and #22  ?

**Comment by OjesviPandita** [comment]
You can ignore PR #22 as this is the latest one.

**Comment by thekirankumar** [comment]
@gauthamsrinivas7 any update on this ?

File: docs/index.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+Ultra is an application development platform within Flipkart application offering an open window for external companies or organizations that are existing or may want to become Flipkart business partners. Ultra will help Flipkart’s business associates in offering a development platform for merging their applications within Flipkart application and provide a variety of services to the customers. It will help Flipkart customers to get access to a single shopping destination. It is an idea designed to pave new ways for users to interact with any external application within Flipkart with security and trust. This can help Flipkart business partners (such as, MakeMyTrip, Phone Pe and so on) in getting closer to one of the largest user base in the country by opening their services like booking tickets, bill payments and others for the users. It will further help Flipkart present an ultimate shopping experience to its customers by launching convenience services such as online ticketing, online payment of bills, so on and by including more business categories indirectly into its domain.
```
Comment (gauthamsrinivas7) [inline docs/index.md]: Possible to separate into "What's in it for partners" and "What's in it for users" sub-sections? Shuttling between the two in the same section is a bit confusing to read.

File: docs/index.md [hunk 2]
```diff
@@ -33,10 +33,11 @@ We have also taken strategic product decisions which has shaped Ultra offering f
 
 # Why Ultra
```
Comment (gauthamsrinivas7) [inline docs/index.md]: Focus more on why ultra for partners here and then bring user angle? End-users won't be reading this document. The partner devs or biz folks reading this should get an idea about what's in it for them

**Pull request merged**